### PR TITLE
Addition of NMOS OAuth security decorators

### PR DESCRIPTION
**Pull request by dannymeloy**
Will protect subscription and web socket endpoints from unauthorized access when the OAUTH_MODE flag in nmoscommon-config is Truthy (default is set to False.)

Also changed imports to explicit relative in line with Python best practices (and so I could run it without installing it initially...)

### CODE REVIEW

**Comment by simonrankine** [comment]
General point - if where you can you can do large linting fixes in one stand-alone commit it would make it easier to see the main changes for the PR ;-)

**Comment by simonrankine** [comment]
General point - do we want to protect all of the endpoints in IS-04, so that baddies can't scrape out information about what's on the network? I thought that was the plan but maybe I didn't communicate that well.

**Comment by dannymeloy** [comment]
I think long term we would probably have everything secured yes, I think this was more of a proof of concept that I could get it working with websockets and limit access to them if a token wasn't present

File: rpm/registryquery.spec [hunk 1]
```diff
@@ -26,7 +26,8 @@ Requires:       python-requests           >= 0.9.3
 Requires:       python-ws4py
 Requires:       ips-etcd
 Requires:       ips-reverseproxy-common
-Requires:	nmoscommon
+Requires:       nmoscommon
+Requires:       python-nmos-oauth
```
Comment (andrewbonney) [inline rpm/registryquery.spec]: As with the other PR(s) I suspect the nmos-oauth dependencies can be removed now

**Comment by andrewbonney** [comment]
A minor thing, but this may be best as a minor version bump rather than a patch.

**Comment by dannymeloy** [comment]
Closing because merge conflicts were resolved in difference branch #35

**Pull request closed without merging**